village/src/App.js

```diff
@@ -14,6 +16,11 @@ class App extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
+  componentDidMount() {
```

Comment:
Great use of a life-cycle method to call the axios get. this will work but think about using a .catch() for error handling